### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SatoCodingStuff**
DO NOT MERGE UNTIL DONE

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## Getting started
+
+Read our README.md and get familiar with the code base! If you need help, join our [discord](https://discord.gg/2cf6b4sdzM) and we will help you.
+
+If can also contribiot in other ways, like opening an issue!
```
Comment (sparkles-laurel) [inline CONTRIBUTING.md]: Typo here: ~~contribiot~~ contribute

Grammar mistake: if *you* can

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Fork the repository.
+- Using GitHub Desktop:
+  - [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/getting-started-with-github-desktop) will guide you through setting up Desktop.
+  - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+- Using the command line:
+  - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
+
+2. Install or update to **Node.js**, at the version specified in `.node-version`. For more information, see [the development guide](contributing/development.md).
```
Comment (sparkles-laurel) [inline CONTRIBUTING.md]: Hm?

Comment (SatoCodingStuff) [reply CONTRIBUTING.md]: Lemme fix that real quick

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Getting started
+
+Read our README.md and get familiar with the code base! If you need help, join our [discord](https://discord.gg/2cf6b4sdzM) and we will help you.
+
+you can also contribute in other ways, like opening an issue!
+
+### Issues
+
+If you are expiriancing some kind of issue or you have an idea, tell us! Open an issue in the issues tab and explain it. Make sure that you are giving all the information you have.
```
Comment (electron271) [inline CONTRIBUTING.md]: typo

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Scan through our [existing issues](https://github.com/github/docs/issues) to find one that interests you. You can narrow down the search using `labels` as filters.
+
+### Make Changes
+
+Please follow our [project manager](https://github.com/orgs/popcorn-kernel/projects/1) and see if someone is working on the changed that you wanna add. Just rememeber that we only apply changed that we see fit.
+
+#### Make changes locally
+
+1. Fork the repository.
+- Using GitHub Desktop:
```
Comment (electron271) [inline CONTRIBUTING.md]: id say only include regular git forking

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+- Using the command line:
+  - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
+
+2. Create a working branch and start with your changes!
+
+### Commit your update
+
+Commit the changes once you are happy with them. Don't forget to self-review to speed up the review process:zap:.
```
Comment (electron271) [inline CONTRIBUTING.md]: remove emoji for consistency

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Pull Request
+
+When you're finished with the changes, create a pull request, also known as a PR.
+- Fill the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request.
```
Comment (electron271) [inline CONTRIBUTING.md]: iirc we do not have templates

**Review by electron271 — CHANGES_REQUESTED** [review]
see all of my comments

**Comment by SatoCodingStuff** [comment]
all fixed

**Pull request merged**